packages/webpack-extraction-plugin/src/webpackLoader.test.ts

```diff
@@ -175,7 +175,7 @@ function testFixture(fixtureName: string, options: CompileOptions = {}) {
     if (expectedError) {
       expect(resultError.message).toMatch(expectedError.default);
     }
-  });
+  }, 10000);
```

Comment:
This test can last longer than the default  5 seconds. Here is a build from main failing: https://github.com/microsoft/griffel/actions/runs/2530419980